### PR TITLE
Add cli command to show the round end summary & add search box to crew manifest

### PR DESCRIPTION
**Pull request by koteq**
## About the PR

- Add a console command to show the round end summary window.
- Add a search box to filter the manifest tab.

It will work if you've seen the round end summary and closed it, it will not work if you joined after the round end.

It may require the engine changes to fix SpriteView rendering in the lobby
- https://github.com/space-wizards/RobustToolbox/pull/4886

## Why / Balance

Dunno, I've heard people keep closing the summary just to realize shortly that they'd like to check something.

## Media

https://github.com/space-wizards/space-station-14/assets/1192090/7a5203e7-7c2c-4e02-9ac3-e7a84f8a1886

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Content.Client/GameTicking/Managers/ClientGameTicker.cs
Comment (metalgearsloth) [inline Content.Client/GameTicking/Managers/ClientGameTicker.cs:186]: Needs localising + dependencies not IOC resolves, also shouldn't be useable by anyone should be behind debug.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Client/GameTicking/Managers/ClientGameTicker.cs
Comment (deltanedas) [reply Content.Client/GameTicking/Managers/ClientGameTicker.cs:186]: why would reopening roundend summary require perms

also command is named Changelog pro

Comment (koteq) [reply Content.Client/GameTicking/Managers/ClientGameTicker.cs:186]: > is named Changelog

Ya, I copy-pasted it from the changelog command. Same goes for the openahelp one. They are all the same.

Comment (koteq) [reply Content.Client/GameTicking/Managers/ClientGameTicker.cs:186]: > Needs dependencies not IOC resolves

I'd argue that it makes no difference in that particular situation. Decorating it and allowing a factory to resolve it is O(1) that involves some reflections shenanigans, and calling IoC resolve, which is backed by a dictionary, also is O(1). Not to mention that it's client-side.

---

Anyway, I'm adding the entity system as a decorated dependency. Can't resolve the ticker right away, as it's unregistered.

> Unhandled exception. Robust.Shared.IoC.Exceptions.UnregisteredDependencyException: Content.Client.GameTicking.Managers.RoundSummaryCommand requested unregistered type with its field Content.Client.GameTicking.Managers.ClientGameTicker: _game

**Comment by VasilisThePikachu** [comment]
Resolves #7567

**Comment by Errant-4** [comment]
Is this still being worked on?

**Comment by koteq** [comment]
@Errant-4, I believe it was ready to be merged several weeks ago.

From my experience working in the repo, it's easy to get a first review from Sloth within a day or two. Unlikely, if you happened to get a changes request, it's nearly impossible to get rid of it. Other maintainers seem to be skipping flagged PRs, while Sloth is rarely paying another visit or answering GH comments.

And since #pr-review-request isn't for merge begging, I just patiently wait for my fate.

**Comment by koteq** [comment]
I won't be able to work on it for a while, feel free to take over.

**Pull request closed without merging**
